apiserver/backrestservice/backrestimpl.go

```diff
@@ -212,9 +214,8 @@ func CreateBackup(request *msgs.CreateBackrestBackupRequest, ns, pgouser string)
 	return resp
 }
 
-func getBackupParams(identifier, clusterName, taskName, action, podName, containerName, backupOpts, backrestStorageType, jobName, ns, pgouser string) *crv1.Pgtask {
+func getBackupParams(identifier, clusterName, taskName, action, podName, containerName, imagePrefix, backupOpts, backrestStorageType, jobName, ns, pgouser string) *crv1.Pgtask {
```

Comment:
Perhaps out of scope for this changeset, however, is it possible to perhaps create some kind of non-exported struct for these params? It just seems cumbersome to work with as it sits.

perhaps something like:

```
type getBackupParamsInfo struct {
  identifier string
  clusterName string,
 ...
}
```

Just a thought to make it a bit easier to maintain/use.

Reply:
Agreed for the purposes of documenting doing these kinds of changes, but I will certainly `echo 'out of scope for this changeset'`

Reply:
I'll plan to leave this as is for the time being, but I am interested in a broader backup design discussion. One of the reasons the `GetImagePrefix` function is written to accept two string values (instead of, for instance, accepting a `pgcluster` CRD is due to the variation in how the values are passed, particularly related to the backups.